package.json

```
@@ -12,11 +12,15 @@
   ],
   "scripts": {
     "build": "webpack -p",
+    "heroku-postbuild": "make emojis && webpack -p --config ./webpack.dev.js",
+    "start": "webpack-dev-server --config ./webpack.dev.js --open",
     "lint": "eslint . --ext .jsx --ext .js",
     "pretest": "npm run lint && npm run prettier",
     "prettier": "prettier --list-different --write \"./**/**.{js,jsx}\"",
-    "test": "jest --coverage --runInBand",
+    "test": "jest --coverage --runInBand --testPathIgnorePatterns '__tests__/browser'",
     "test.watch": "jest --watch --coverage=false",
+    "test.browser": "jest --runInBand --testPathPattern '__tests__/browser'",
```

Should we prefix this with the Jest cache busting command you showed me? (I recently found out you can add a `pre` and `post` tasks for _any_ script and NPM will automatically execute 'em in order when you run the main task!)

```suggestion
    "pretest.browser": "jest --clearCache",
    "test.browser": "jest --runInBand --testPathPattern '__tests__/browser'",
```

---

oooh TIL. very cool.